### ISSUE TITLE
Disable submit button when the signatures are not enough

### ISSUE THREAD

**Issue by siegfried** [ADAOcommunity/round-table#44]
Not a big issue but it might be good to prevent spam submission?

**Comment by thenic95** [comment]
Yeah that would be good. Same as when you create a transaction and the **Review Transaction** button is greyed out.

![image](https://user-images.githubusercontent.com/61423239/162947435-e50f3063-ea53-47be-8b5f-7340ec67b29b.png)

**Comment by cherrries** [comment]
I support this feature as being useful for reducing confusion for new and non-technical users.
